app/code/Magento/BraintreeGraphQl/etc/schema.graphqls

```diff
@@ -11,9 +11,9 @@ input PaymentMethodInput {
 }
 
 input BraintreeInput {
-    payment_method_nonce: String!
-    is_active_payment_token_enabler: Boolean!
-    device_data: String
+    payment_method_nonce: String! @doc(description:"Required input for Braintrees payments")
+    is_active_payment_token_enabler: Boolean! @doc(description:"Required input for Braintrees payments")
```

Comment:
This field is required only if Vault is enabled for Braintree payment integration. The `is_active_payment_token_enabler` shows if an entered by a customer credit/debit card should be tokenized for later usage.